build/verify-features.ts

```diff
@@ -0,0 +1,47 @@
+import path from 'node:path';
+import {readdirSync as readDirSync, readFileSync} from 'node:fs';
```

Comment:
No need to rename code APIs

```suggestion
import {readdirSync, readFileSync} from 'node:fs';
```